sippy-ng/src/App.js

```diff
@@ -288,116 +288,122 @@ export default function App(props) {
 
                     <Route
                       path="/release/:release/tags/:tag"
-                      render={(props) => (
+                      render={(theProps) => (
```

Comment:
To be honest I don't like this, js has scopes for a reason and shadowing variables outside your scope you're not going to use doesn't seem problematic to me.  Is it possible to just turn on the global shadowing? I think that's golang's default behavior and I find it not too intrusive.

Reply:
I hear ya.  I started out by doing a bunch of `/* eslint-{en|dis}able no-shadow */` around those and then maybe changing the name of `props` at the top (one line change) vs. `theProps` everywhere which I've never seen on any example code.  I didn't see much in the way of making this more forgiving in [the doc](https://eslint.org/docs/latest/rules/no-shadow#rule-details).  I also noticed that doc is for v8.52.0 and we're running v7.32.0 so some options don't work.  I might try upgrading eslint and see what happens.

I definitely would like to eliminate the changes regarding the local scope stuff esp. for things like in the `then` clauses in promises and functions with `event`.